brittonjg/britton.app
# Portfolio v1

## Purpose
Resolves #3 

## Screenshots
Following is an example of before and after development

### Web
Before             |  After
:-------------------------:|:-------------------------:
![Web - Before](https://user-images.githubusercontent.com/12110204/50864673-b12e3e80-139a-11e9-8bf9-bf33fab5fc22.jpg) | ![Web - After](https://user-images.githubusercontent.com/12110204/50864612-817f3680-139a-11e9-9422-f5ed6c6bc315.png)

### Mobile
Before             |  After
:-------------------------:|:-------------------------:
![Mobile - Before](https://user-images.githubusercontent.com/12110204/50864673-b12e3e80-139a-11e9-8bf9-bf33fab5fc22.jpg) | ![Mobile - After](https://user-images.githubusercontent.com/12110204/50864614-817f3680-139a-11e9-8aa3-46ba7c40fef1.png)

## Testing
#### Steps
How to specifically test the changes that have been made:
1. **VERIFY** that all links and content loads correctly
2. **VERIFY** that most screen sizes are tested

## Other Info
None.
